dissect/target/plugins/os/unix/cronjobs.py

```diff
@@ -29,82 +34,128 @@
-                cmd = match.group(7)
-                if not str(file_path).startswith("/etc/crontab") or not str(file_path).startswith("/etc/cron.d"):
-                    cmd = usr + " " + cmd
-                    usr = ""
-
-                yield CronjobRecord(
-                    minute=match.group(1),
-                    hour=match.group(2),
-                    day=match.group(3),
-                    month=match.group(4),
-                    weekday=match.group(5),
-                    user=usr,
-                    command=cmd,
-                    source=file_path,
-                    _target=self.target,
-                )
-
-            s = re.search(r"^([a-zA-Z_]+[a-zA-Z[0-9_])=(.*)", line)
-            if s:
-                yield EnvironmentVariableRecord(
-                    key=s.group(1),
-                    value=s.group(2),
-                    source=file_path,
-                    _target=self.target,
-                )
+            for file in dir.iterdir():
+                if file.resolve().is_file():
+                    yield file
+
+        for crontab_file in self.CRONTAB_FILES:
+            if (file := self.target.fs.path(crontab_file)).exists():
+                yield file
 
     @export(record=[CronjobRecord, EnvironmentVariableRecord])
     def cronjobs(self) -> Iterator[CronjobRecord | EnvironmentVariableRecord]:
-        """Yield cronjobs on the unix system.
+        """Yield cronjobs on a unix system.
 
         A cronjob is a scheduled task/command on a Unix based system. Adversaries may use cronjobs to gain
         persistence on the system.
-        """
-        tabs = []
-        crontab_dirs = [
-            "/var/cron/tabs",
-            "/var/spool/cron",
-            "/var/spool/cron/crontabs",
-            "/etc/cron.d",
-            "/usr/local/etc/cron.d",  # FreeBSD
-        ]
-        for path in crontab_dirs:
-            fspath = self.target.fs.path(path)
-            if not fspath.exists():
-                continue
 
-            for f in fspath.iterdir():
-                if not f.exists():
-                    continue
-                if f.is_file():
-                    tabs.append(f)
+        Resources:
+            - https://linux.die.net/man/8/cron
+            - https://linux.die.net/man/1/crontab
+            - https://linux.die.net/man/5/crontab
+            - https://en.wikipedia.org/wiki/Cron
+            - https://linux.die.net/man/8/anacron
+            - https://manpages.ubuntu.com/manpages/oracular/en/man5/crontab.5.html
+            - https://www.gnu.org/software/mcron/manual/mcron.html#Guile-Syntax
+        """
 
-        crontab_file = self.target.fs.path("/etc/crontab")
-        if crontab_file.exists():
-            tabs.append(crontab_file)
+        for file in self.crontabs:
+            # Cronjobs in user crontab files do not have a user field specified.
+            user = None
+            if file.is_relative_to("/var/spool/cron/crontabs"):
```

Comment:
Did some small testing on a Fedora 41 machine. Some cronjobs also seem to get created in `/var/spool/cron/<username>`
```suggestion
            if file.is_relative_to("/var/spool/cron/crontabs") or file.is_relative_to("/var/spool/cron/"):
```